src/Uno.Extensions.Navigation.UI/Controls/ForceUpdateAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Uno.Extensions.Navigation.UI.Controls;
+
+/// <summary>
+/// Flags the default constructor to use to create an instance of a record that is being de-normalized for bindings.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class)]
+public class ForceUpdateAttribute : Attribute
+{
+	/// <summary>
+	/// Gets or sets a value indicating whether the method should be generated or not.
+	/// </summary>
+	public bool IsEnabled { get; set; } = true;
+
+	/// <summary>
+	/// Configure generation the force update method.
+	/// </summary>
+	/// <param name="isEnabled">Indicates if the method should be generated or not.</param>
+	public ForceUpdateAttribute(bool isEnabled = true)
+	{
+		IsEnabled = isEnabled;
+	}
```

Comment:
I wouldn't introduce the same thing both as constructor argument and named argument. Only one should be needed, and actually having both is confusing IMO.

Reply:
yep, will make property read only